src/world/World.js

```diff
@@ -520,9 +520,9 @@ var step_tmp1 = new Vec3();
  */
 World.prototype.step = function(dt, timeSinceLastCalled, maxSubSteps){
     maxSubSteps = maxSubSteps || 10;
-    timeSinceLastCalled = timeSinceLastCalled || 0;
+    if (typeof timeSinceLastCalled === 'undefined') timeSinceLastCalled = -1;
 
-    if(timeSinceLastCalled === 0){ // Fixed, simple stepping
+    if(timeSinceLastCalled === -1){ // Fixed, simple stepping
```

Comment:
Or better yet, let's not define an exception value at all. Let's have:

`if(typeof timeSinceLastCalled === 'undefined'){ // Fixed, simple stepping`